docs/examples/4_ballast_movements.py

```diff
@@ -10,12 +10,8 @@
     search_result = VesselMovements().search(
         filter_time_min=datetime(2017, 10, 1, 0),
         filter_time_max=datetime(2017, 10, 1, 1),
+        filter_vessel_status="vessel_status_ballast",
```

Comment:
Just to incorporate the new filter instead of filtering on product retrospectively.